WalletWasabi.Fluent/ViewModels/Dialogs/SpeedUpTransactionDialogViewModel.cs

```diff
@@ -0,0 +1,90 @@
+using System.Linq;
+using System.Reactive;
+using System.Threading.Tasks;
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Blockchain.TransactionBuilding;
+using WalletWasabi.Blockchain.Transactions;
+using WalletWasabi.Fluent.Extensions;
+using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
+using WalletWasabi.Logging;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.ViewModels.Dialogs;
+
+[NavigationMetaData(Title = "Speed Up Transaction")]
+public partial class SpeedUpTransactionDialogViewModel : DialogViewModelBase<Unit>
+{
+	private readonly Wallet _wallet;
+
+	private SpeedUpTransactionDialogViewModel(Wallet wallet, SmartTransaction transactionToSpeedUp, BuildTransactionResult boostingTransaction)
+	{
+		_wallet = wallet;
+		SetupCancel(enableCancel: true, enableCancelOnEscape: true, enableCancelOnPressed: true);
+
+		EnableBack = false;
+		NextCommand = ReactiveCommand.CreateFromTask(() => OnSpeedUpTransactionAsync(boostingTransaction));
+
+		FeeDifference = GetFeeDifference(transactionToSpeedUp, boostingTransaction);
```

Comment:
Is the `FeeDisplayUnit` setting handled here?

Reply:
Yes, it's using a converter in the XAML that takes care of everything: 

<TextBlock TextAlignment="Center"
                       FontWeight="Bold"
                       Text="{Binding FeeDifference, Converter={x:Static converters:MoneyConverters.ToFeeWithUnit}, FallbackValue='0.0000 0232 BTC'}"
                       Classes="h5" VerticalAlignment="Bottom" />